lib/Doctrine/DBAL/Migrations/Migration.php

```diff
@@ -95,14 +95,18 @@ public function writeSqlFile($path, $to = null)
 
         $this->outputWriter->write(sprintf("-- Migrating from %s to %s\n", $from, $to));
 
-        $sqlWriter = new SqlFileWriter(
+        return $this->createSqlFileWriter($path)
+                    ->write($sql, $direction);
+    }
+
+    protected function createSqlFileWriter(string $path): SqlFileWriter
```

Comment:
Which now is https://github.com/doctrine/migrations/pull/537/files#diff-5a519f363afe6bb6f788510621f527f0R188